lint-js.sh

```diff
@@ -3,9 +3,19 @@
 # Run setup.
 . ./setup.sh
 
-# Run the lint command from the wp-content directory.
-if [ "$fix" = "1" ]; then
-	npm run lint:js "$plugindir" -- --config .eslintrc --fix;
-else
-	npm run lint:js "$plugindir" -- --config .eslintrc;
+# Check if there is an .eslintrc file in this project. If so, use it for standards. Otherwise,
+if [ -f "$plugindir/.eslintrc" ];
+then
+	if [ "$fix" = "1" ]; then
+		npm run lint:js "$plugindir" -- --config $plugindir/.eslintrc --fix;
+	else
+		npm run lint:js "$plugindir" -- --config $plugindir/.eslintrc;
+	fi
+else 
+	# Run the lint command from the wp-content directory.
+	if [ "$fix" = "1" ]; then
+		npm run lint:js "$plugindir" -- --fix;
```

Comment:
`--config .eslint` might still be needed here:

```diff
diff --git a/lint-js.sh b/lint-js.sh
index dcafbd4..3b15ce8 100644
--- a/lint-js.sh
+++ b/lint-js.sh
@@ -14,8 +14,8 @@ then
 else 
        # Run the lint command from the wp-content directory.
        if [ "$fix" = "1" ]; then
-               npm run lint:js "$plugindir" -- --fix;
+               npm run lint:js "$plugindir" -- --config .eslintrc --fix;
        else
-               npm run lint:js "$plugindir";
+               npm run lint:js "$plugindir" -- --config .eslintrc;
        fi
 fi
```

Without it, there are a lot of errors like:

```
pattern-manager/wp-modules/pattern-post-type/js/src/hooks/usePatternData.ts
  1:1  error  Parsing error: The keyword 'import' is reserved
```

Steps to reproduce:
1. `cd wp-content`
2. `rm -rf wpps-scripts`
3. `git clone -b try/loading-config-files-from-plugin https://github.com/wp-plugin-sidekick/wpps-scripts
4. `cd plugins/pattern-manager`
5. `npm run lint:js`
6. Expected: it passes
7. Actual: It shows errors above

This [test PR](https://github.com/studiopress/pattern-manager/pull/8) of this branch also shows those errors. Hope it's OK I opened that.

Reply:
Thanks!

Reply:
Also, did the 'Steps to reproduce' help? I'm trying to balance giving enough information, and being too verbose.